www/nginx/Portfile

```diff
@@ -104,6 +104,15 @@ post-destroot {
     }
 
     file rename ${destroot}${prefix}/html ${destroot}${nginx_share}
+
+    # Install the manpage
+    set man_path "${destroot}${prefix}/share/man/man8"
+    set gzip [findBinary gzip ${portutil::autoconf::gzip_path}]
+
+    xinstall -d -m 0755 ${man_path}
+    xinstall    -m 0644 ${worksrcpath}/man/${name}.8 ${man_path}
+    reinplace -q "s|/var/run/mynginx.pid|${nginx_pidfile}/|g" ${man_path}/${name}.8
+    system "$gzip -9vf ${man_path}/${name}.8"
```

Comment:
There is no need to manually compress the manpage – MacPorts will automatically compress manpages installed in `$prefix/share/man`.

I suggest you remove the line that sets `$gzip` and the invocation of `system` for this reason.

Reply:
Ah, ok… I thought it was needed. I “borrowed” that gzip line from the mariadb port. [»»](https://github.com/macports/macports-ports/blob/master/databases/mariadb-10.5/Portfile#L253)

Done √